### ISSUE TITLE
PIPAL: a Large-Scale Image Quality Assessment Dataset for Perceptual Image Restoration 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#9]
# TL;DR
They proposed an evaluation method using PIPAL, a dataset that includes various types of distortions such as GAN generation, and evaluate common metrics by Elo rating, which is often used to describe the strength of chess and other games. Evaluating the commonly used PSNR and SSIM using it, They found that they are incompatible with the perceptual results, other metrics also seem to need improvement. 	 	 	

# Why it matters:

# Paper URL
https://arxiv.org/abs/2007.12142

# Submission Dates(yyyy/mm/dd)
2020/07/23 

# Authors and institutions
Jinjin Gu, Haoming Cai, Haoyu Chen, Xiaoxing Ye, Jimmy Ren, Chao Dong
 - The Chinese University of Hong Kong 	
- SIAT-SenseTime  	
- SenseTime Research 	
- Shenzhen Institute of Artificial Intelligence and Robotics for Society 

# Methods

# Results

# Comments
ECCV2020
